frontend/src/pages/accounts/[id].tsx

```diff
@@ -119,7 +119,7 @@ export const getServerSideProps: GetServerSideProps<
     return {
       props: {
         account: { accountId: id },
-        accountError,
+        accountError: String(accountError),
```

Comment:
[There](https://github.com/near/near-explorer/blob/2fb503266e5ea5f7b60b416a1d046fbaa6a01484/frontend/src/pages/accounts/%5Bid%5D.tsx#L113) must be the same

Reply:
Yup, you're right

Reply:
@shelegdmitriy Need your approve now :)